### DISCUSSION TITLE
Can't access unencrypted local SMTP server

### DISCUSSION THREAD

**Discussion by KikoPT1234** [pocketbase/pocketbase#3751] [Q&A]
I'm running pocketbase in a Docker container and a Postfix instance in another one. This instance is really minimal - it just serves as a no-reply email sender for transactional emails, and it's closed off to the exterior, so I don't need to implement TLS/SSL encryption. I'm able to use Postfix with SMTP using nodemailer with TLS off, however if I try to use PocketBase, it always gives one of these errors:
```
x509: certificate is valid for localhost, not postfix.
// or
Raw error: unencrypted connection.
```

I noticed that the second error happened after I set smtp_tls_security_level to none in Postfix, however after reverting the change PocketBase still gives the second error, suggesting that there is some under-the-hood caching going on.
I apologize if there is something obvious that I'm missing.

PS: Yes, I did try both `Auto (StartTLS)` and `Always` in the TLS encryption option.

**Comment by ganigeorgiev** [comment]
I'm not sure what could be the cause of the error. Double check that the smtp host matches with your mail server.

I guess https://github.com/pocketbase/pocketbase/discussions/669 could be also related, but I'm still not sure that adding the [`InsecureSkipVerify`](https://pkg.go.dev/crypto/tls#Config.InsecureSkipVerify) option would be a good idea.

For quick email tests I often use and recommend the dummy smtp service of [mailtrap.io](https://mailtrap.io/email-sandbox/) (they have free plan and a convenient email preview; they even recently started offering free [Email sending](https://mailtrap.io/email-sending/) with 1k/month emails).
